disperser/common/chunkstore/chunk_metadata_store.go

```diff
@@ -0,0 +1,42 @@
+package chunkstore
+
+import (
+	commondynamodb "github.com/Layr-Labs/eigenda/common/aws/dynamodb"
+	"github.com/Layr-Labs/eigenda/disperser"
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"time"
+)
+
+// ChunkMetadataStore is an interface for storing and retrieving metadata about chunks.
+type ChunkMetadataStore interface {
```

Comment:
Not sure if we need another store abstraction for writing/reading chunk metadata. 
Since chunk metadata lives inside blob metadata, I think write/read should happen via [blob metadata store](https://github.com/Layr-Labs/eigenda/pull/846/files#diff-e7315f00f40f5ec590e1c7e8b6a46fb07d214931d8ea9d776fd57266dd6e7e39R74).

Reply:
This was something separate since we had originally discussed not putting the extra metadata into the regular blob metadata store. Now that this data has merged into the other blob matadata, I agree that it doesn't make sense to have a separate chunk metadata store. Removed.